lib/ReactViews/ExplorerWindow/Tabs/DataCatalogTab.tsx

```diff
@@ -57,21 +58,22 @@ class DataCatalogTab extends React.Component {
             <Box className={Styles.dataExplorer} styledWidth="40%">
               {/* ~TODO: Put this back once we add a MobX DataCatalogSearch Provider~ */}
               {/* TODO2: Implement a more generic MobX DataCatalogSearch */}
-              {searchState.catalogSearchProvider && (
+              {state.catalogSearchProvider ? (
                 <SearchBox
-                  searchText={searchState.catalogSearchText}
-                  onSearchTextChanged={(val) => this.changeSearchText(val)}
+                  searchText={state.catalogSearchText}
+                  onSearchTextChanged={(val: string) =>
+                    this.changeSearchText(val)
+                  }
                   onDoSearch={() => this.search()}
                   placeholder={this.searchPlaceholder}
                   debounceDuration={
-                    terria.catalogReferencesLoaded &&
-                    searchState.catalogSearchProvider
-                      ? searchState.catalogSearchProvider
-                          .debounceDurationOnceLoaded
+                    terria.catalogReferencesLoaded
```

Comment:
There is already a check of `state.catalogSearchProvider` on line 61, so removed the check on this line.

Also had to shorten `searchState` to get pretty to keep the `@ts-expect-error` in the right place.